src/dashboard/src/components/api/views.py

```diff
@@ -113,7 +113,10 @@ def get_unit_status(unit_uuid, unit_type):
     :return: Dict with status info.
     """
     ret = {}
-    job = models.Job.objects.filter(sipuuid=unit_uuid).filter(unittype=unit_type).order_by('-createdtime', '-createdtimedec')[0]
+    job = models.Job.objects.filter(
+        sipuuid=unit_uuid).filter(
+            unittype=unit_type).order_by(
+                '-createdtime', '-createdtimedec')[0]
```

Comment:
Holly and I have been debating this for quite some time. I'm more of a stickler for PEP 8's "Limit all lines to a maximum of 79 characters", cf. https://www.python.org/dev/peps/pep-0008/#maximum-line-length. I have a vim shortcut to make breaking lines easier. And pylint complains about 79-char+ lines by default. And I kinda think you should be able to print your code on a piece of a paper and have it be readable.

That said, pretty much all of the AM codebase ignores this 79 character rule, so being fully consistent would requite a LOT of pointless work. And modern editors can make your long lines quite readable. So I'm open to undoing the above change.

A third option would be for me (and other 79-character sticklers) to not change existing code but continue to insert linebreaks in new code. I can make my editor make long lines readable while still inserting linebreaks in my contributions. I think on this one point of style we can have two opinions/styles co-existing, so this is what I would prefer. Thoughts?